main.py

```diff
@@ -32,6 +32,9 @@ def main():
             next_guess = console.input("Enter your next guess:-> ").upper()
 
             try:
+                if next_guess.upper() not in word_list:
+                    raise InvalidEntryError(f"Your guess '{next_guess}' is not a valid word.  Please try again.")
+
```

Comment:
Since you're at the top level, the raise is unnecessary.  I think you can just:

```suggestion
                   console.print(f"Your guess '{next_guess}' is not a valid word.  Please try again.")
                   continue

```

Reply:
Actually, disregard my comment. If it works this way, I think your changes throughout this entire pull request gave me a better idea to represent the word list, and this change highlights the need.  I'll explain more off line.